styleguide/source/assets/js/block-select.js

```diff
@@ -0,0 +1,9 @@
+jQuery.noConflict();
+(function($) {
+  $('.block-select__message').hide();
+
+  $('.block-select').click(function() {
+    $(this).toggleClass('block-select--selected');
+    $('.block-select__message').toggle();
+  });
+})(jQuery);
```

Comment:
Technically, this is fine. But in general I dislike using jQuery's show and hide because they inject inline styles into the DOM, whereas swapping classes for show/hiding is more efficient. So might I suggest the following approach:
 - Remove line 3 (this element should be hidden by the css when the page loads rather than hidden by js on load anyway
 - Change line 7 to something like `$('.block-select__message').toggleClass('block-select__message--hidden');`
 - See the css above for suggestions as to how that needs to be modified to make this work.